handbook/ce/customer-support-onboarding.md

```diff
@@ -0,0 +1,5 @@
+# Customer Support Onboarding (WIP)
+
+Currently a work-in-progress; expected to have a working version 1 by 2021-02-19
+
+Welcome to Sourcegraph, to the Customer Engineering department, and the Customer Support team! This document will guide you through customer support-specific onboarding. It builds on our company onboarding and focuses on everything you need to be successfull in this particular role.
```

Comment:
@virginiaulrich Should that be "successful" one l?

Reply:
@emchap yup! Updated :)